src/Main.java

```diff
@@ -1,42 +1,57 @@
+        String[] alphabetic = {"Een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
+        //System.out.println(numeric);
+        //Nu ga je deze aanroepen met de juiste argumenten in de main methode van de Main class, oftewel: maak in main een nieuw object aan van het type Translator.
+        Translator translator = new Translator(numeric, alphabetic);
 
-        // Hier mag je je code scrijven voor de hoofd-opdracht
+        //Maak in de main methode van de Main class een boolean variabele genaamd play
+        // met de waarde true. Maak een String variabele genaamd ongeldig waarin je de String "ongeldige invoer" zet.
+        // Als laatst maak je nog een nieuw Scanner object aan met Scanner scanner = new Scanner(System.in).
 
-        /* deze regel mag je weg halen voor de bonus opdracht. Onderstaande code is voor de bonus opdracht.
-        HashSet<Integer> secretnumber = randomnumbergenerator();
-        String stringnumber =  setToStringConverter(secretnumber);
-        System.out.println(stringnumber);
-        feedback();
-         deze regel mag je weg halen voor de bonus opdracht */
-
-    }
-
-    /*
-     Deze methode is voor de bonus opdracht.
-     */
-    public static void feedback(String stringnumber) {
+        Boolean play = true;
+        String ongeldig = "Ongeldige invoer";
         Scanner scanner = new Scanner(System.in);
-        StringBuilder feedback = new StringBuilder();
-        System.out.println("take a guess");
-        String guess = scanner.nextLine();
-        if (Objects.equals(guess, stringnumber)) {
-            System.out.println("gefeliciteerd je hebt het goed");
-        } else {
-            for (int i = 0; i < 4; i++) {
-                if (guess.substring(i, i + 1).equals(stringnumber.substring(i, i + 1))) {
-                    feedback.append("+");
-                } else if (stringnumber.contains(guess.substring(i, i + 1))) {
-                    feedback.append("0");
-                } else {
-                    feedback.append("X");
+        //Vervolgens maak je een while-loop die doorgaat zolang play true is.
+        while(play){
+            //Print in de while-loop "Type 'x' om te stoppen \nType 'v' om te vertalen" en maak een String variabele genaamd input waarin je de waarde van scanner.nextLine() opslaat.
+            System.out.println("Type 'x' om te stoppen");
+            System.out.println("Type 'v' om te vertalen");
+            String input = scanner.nextLine();
+            //i. if: Als de input "x" is, dan zet je play naar false.
+            if(Objects.equals(input, "x")){
+                play = false;
+            }
+            //ii. else if: Als de input "v" is, dan print je eerst "Type een cijfer in van 0 t/m 9",
+            else if (Objects.equals(input, "v")) {
```

Comment:
Het werkt dus dat is top! Maar dit zou je wel korter kunnen schrijven als: `input.equals("x")`